test/test_git_helpers.py

```diff
@@ -0,0 +1,43 @@
+import time
+
+import conda_build_prepare.git_helpers as gh
+
+COMMIT_DELAY = 1
+
+
+def git_commit_and_tag(filepath, msg, tag=None):
+    now = time.time()
+    with open(filepath, 'w') as f:
+        f.write(f'{now}')
+    gh._call_custom_git_cmd(filepath.parent, f'add {filepath}')
+    gh._call_custom_git_cmd(filepath.parent, f'commit -m {msg}')
+    if tag:
+        gh._call_custom_git_cmd(filepath.parent, f'tag {tag}')
+    time.sleep(COMMIT_DELAY)
```

Comment:
You might need to explain why you need `time.sleep` and `COMMIT_DELAY`?

Reply:
otherwise git get confused about the sorting with all the commit dates being the same (it doesn't got below second resolution).

I'll add a comment.